runtime/Cargo.toml

```diff
@@ -21,8 +21,8 @@ deno_fetch = "0.151.0"
 deno_url = "0.127.0"
 deno_web = "0.158.0"
 deno_webidl = "0.127.0"
-lassie = "0.9.0"
-# lassie = { git = "https://github.com/filecoin-station/rusty-lassie.git" }
+# lassie = "0.9.0"
+lassie = { git = "https://github.com/filecoin-station/rusty-lassie.git" }
```

Comment:
AFAICT, the latest published version is 0.10.0 

https://crates.io/crates/lassie

Reply:
Upgraded in https://github.com/CheckerNetwork/zinnia/pull/668/commits/29c9fd61ae221182f041c837965fb5c3bc80b5f8. This was a workaround I've used to test compilation with new version until it got published to crates.io :)